xbmc/platform/android/activity/XBMCApp.h

```diff
@@ -258,7 +259,7 @@ class CXBMCApp
   static void SetRefreshRateCallback(CVariant *rate);
   static void SetDisplayModeCallback(CVariant *mode);
 
-  void RegisterDisplayListener();
+  static void RegisterDisplayListener(CVariant*);
```

Comment:
https://github.com/xbmc/xbmc/blob/af25e3cc809730446f3739ed2ab9cc42a1a3eda9/xbmc/platform/android/activity/XBMCApp.cpp#L388

whoops sry, i thought you would fully automated change the definition to
`void CXBMCApp::RegisterDisplayListener(CVariant*);` ;)

Reply:
Ooops, sorry.